### PR TITLE
fixes #31 and implements officer

### LINKED ISSUE #31: Create a separate pages for rate and rate update
Can you please create separate views for the rates and rate update for now? Our files were merged together in your pull request.

### PR DESCRIPTION
**Pull request by sassansh**
fixes #31 and implements officer

### CODE REVIEW

File: templates/rateUpdateTemplate.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php include 'inc/header.php'; ?>
+<form method="post" action="rateUpdate.php">
+<div class="form-group">
+        <label>Choose Rate Type</label>
+        <select name="rateTypeChange" class="form-control">
+            <option value="0"> </option>
+            <option value="Charging">Charging</option>
```
Comment (itstonywu) [inline templates/rateUpdateTemplate.php]: we should do a foreach loop here over the ratetype table instead of hardcoding it

**Review by itstonywu — CHANGES_REQUESTED** [review]
Please use a foreach over the options for rate type

**Review by Junkim97 — APPROVED** [review]

File: templates/rateUpdateTemplate.php (hunk 1 above)
Comment (sassansh) [reply templates/rateUpdateTemplate.php]: fixed!

**Comment by itstonywu** [comment]
thanks @sassansh

**Review by itstonywu — APPROVED** [review]
lgtm!

**Comment by itstonywu** [comment]
@sassansh go ahead and merge your branch

**Pull request merged**